src/components/Input/Input.module.css

```diff
@@ -18,15 +18,15 @@
 
 .input:focus {
   outline: none;
-  box-shadow: 0 0 0 4px var(--color-primary-light);
+  box-shadow: 0 0 0 2px var(--color-primary-contrast);
```

Comment:
Can we revert it back to 4px and light color (also for the button). I know its different from the design but it looks better I think.

I need to update the design... 😑